source/common/stats/stats_impl.h

```diff
@@ -366,7 +399,21 @@ class HistogramImpl : public Histogram, public MetricImpl {
   // Stats::Histogram
   void recordValue(uint64_t value) override { parent_.deliverHistogramToSinks(*this, value); }
 
+  void merge() override {}
```

Comment:
Does this ever get called? If not I would do `NOT_IMPLEMENTED`. Same for used(). It would probably be better to actually split the histogram interfaces into two different interfaces to avoid this. (Basically some type of parent interface, and then a basic histogram interface). If you don't want to do that I would do NOT_IMPLEMENTED and put in a TODO.